### PR TITLE
Implemented project-iam solution

### PR DESCRIPTION
**Pull request by trodge**

### CODE REVIEW

File: config-connector/solutions/iam/kpt/project-iam/Kptfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: project-iam
+packageMetadata:
+  shortDescription: enable permissions for a project
```
[the remark this answers is not in the document]
Comment (trodge) [reply config-connector/solutions/iam/kpt/project-iam/Kptfile]: I have made this change.

File: config-connector/solutions/iam/kpt/project-iam/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Project IAM
+==================================================
+
+# NAME
+  project-iam
+# SYNOPSIS
+  Config Connector compatible YAML files to enable a role for a member in a project.
```
[the remark this answers is not in the document]
Comment (trodge) [reply config-connector/solutions/iam/kpt/project-iam/README.md]: Change made.

File: config-connector/solutions/iam/kpt/project-iam/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/project-iam project-iam
+  ```
+# USAGE
+## SETTERS
+|    NAME    |        VALUE         |     SET BY      |       DESCRIPTION        | COUNT |
+|------------|----------------------|-----------------|--------------------------|-------|
+| member     | ${IAM_MEMBER?}       | PLACEHOLDER     | IAM member to grant role | 1     |
+| project-id | ${PROJECT_ID?}       | PLACEHOLDER     | ID of project            | 1     |
+| role       | roles/logging.viewer | package-default | IAM role to grant        | 1     |
+
```
[the remark this answers is not in the document]
Comment (trodge) [reply config-connector/solutions/iam/kpt/project-iam/README.md]: I have instead moved this table to its own section.

File: config-connector/solutions/iam/kpt/project-iam/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+Setters marked as `PLACEHOLDER` are required. Set them using kpt:
+```
+kpt cfg set . member user:name@example.com
+kpt cfg set . project-id your-project
+```
+Setters set by `package-default` have a default value which can be changed in the same manner.
```
[the remark this answers is not in the document]
Comment (trodge) [reply config-connector/solutions/iam/kpt/project-iam/README.md]: Added "optionally"

File: config-connector/solutions/iam/kpt/project-iam/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (maqiuyujoyce) [reply config-connector/solutions/iam/kpt/project-iam/README.md]: Discussed offline and agreed that we will move the setter table out of usage section.

File: config-connector/solutions/iam/kpt/project-iam/Kptfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (maqiuyujoyce) [reply config-connector/solutions/iam/kpt/project-iam/Kptfile]: Thank you for the confirmation. Next time, if you've believed that you've addressed the comment correctly, please respond in the comment and resolve it.

**Pull request merged**